extensions/UserStory/Extension.pm

```diff
@@ -68,11 +68,15 @@ sub bug_update_before_logging {
   my ($self, $args) = @_;
   my $changes = $args->{changes};
   return unless exists $changes->{cf_user_story};
-  my $diff = diff(
-    \$changes->{cf_user_story}->[0],
-    \$changes->{cf_user_story}->[1],
-    {CONTEXT => 0,},
-  );
+
+  my $old_value = $changes->{cf_user_story}->[0];
+  my $new_value = $changes->{cf_user_story}->[1];
```

Comment:
If the newline is stripped by the time it reaches this variable in Perl, could the issue be that the JS on the front-end is stripping the newline? Fixing it there is probably the more correct solution, but this works too and the bug probably doesn't warrant much more of a time investment. :)